0.download_data/README.md

```diff
@@ -0,0 +1,13 @@
+# Download images from each plate from the NF1 Schwann cell project
+
+In this module, we download the images and platemap metadata for each plate from [Figshare](https://figshare.com/), which is a open-source repository that holds data/figures/etc. to preserve and share. 
+
+## Step 1: Access the project on Figshare
+
+To access the plates, go to the [NF1 Schwann Cell Genotype Cell Painting Assay](https://figshare.com/projects/NF1_Schwann_Cell_Genotype_Cell_Painting_Assay/161620).
+
+## Step 2: Download the images and metadata from each plate
```

Comment:
Consider a programmatic download! (Rather than manual)

Programmatic downloads are far preferable. See https://github.com/gwaybio/drug_viability_data/blob/download-utils/utils/download_utils.py#L6 for an example (you'll likely need to modify this)

Reply:
Working on this now! 

My plan is to have it download the files into a zip, extract files from the zip and remove the zip file, then move the metadata files for both plates into a new metadata directory.